source/common/network/tcp_listener_impl.cc

```diff
@@ -127,5 +128,10 @@ void TcpListenerImpl::setRejectFraction(const UnitFloat reject_fraction) {
   reject_fraction_ = reject_fraction;
 }
 
+void TcpListenerImpl::configureLoadShedPoints(
+    Server::LoadShedPointProvider& load_shed_point_provider) {
+  listener_accept_ = load_shed_point_provider.getLoadShedPoint("core.listener_accept");
```

Comment:
Since this load shed point is TCP specific, should we make the name `core.tcp_listener_accept`, just to be clear?

Reply:
+1.
Maybe there will be something similar for UDP down the line.